src/main/resources/driven-adapter/mongo-reactive/config/mongo-config.java.mustache

```diff
@@ -6,12 +6,14 @@ import co.com.bancolombia.secretsmanager.api.GenericManagerAsync;
 import org.springframework.beans.factory.annotation.Value;
 {{/include-secret}}
 import {{package}}.mongo.config.MongoDBSecret;
-import org.springframework.boot.autoconfigure.mongo.MongoClientFactory;
-import org.springframework.boot.autoconfigure.mongo.MongoProperties;
+import org.springframework.boot.autoconfigure.mongo.*;
```

Comment:
you required all class from org.springframework.boot.autoconfigure.mongo?

Reply:
When a project imports three or more classes from the same package the IDE optimize imports like this.